src/main/java/ua/biz/test/config/Application.java

```diff
@@ -0,0 +1,88 @@
+package ua.biz.test.config;
+
+import ua.biz.test.data.DBCredential;
+import ua.biz.test.data.TableAccess;
+import ua.biz.test.pojo.Employee;
+import ua.biz.test.pojo.JobVacancy;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Application {
+    private List<Employee> employees;
+
+    public Application() {
+        employees = new ArrayList<>();
+    }
+
+    public void getEmployee() {
+        try (Connection con = DriverManager.getConnection(TableAccess.URL, DBCredential.NAME, DBCredential.PASSWORD);
+             Statement stmt = con.createStatement();
+             ResultSet rs = stmt.executeQuery("SELECT * FROM " + TableAccess.TABLE1)) {
+
+            while (rs.next()) {
+                Employee employee = new Employee();
+                employee.setNumber(rs.getString(1));
+                employee.setFirstName(rs.getString(4));
+                employee.setLastName(rs.getString(3));
+
+                employees.add(employee);
+                System.out.println(employee);
+            }
+        } catch (SQLException e) {
+            System.out.println("employee " + e.getMessage());
+        }
+    }
+
+    public void getVacancy() {
```

Comment:
u better split u'r fetching to the different sql data providers

Reply:
fixed